pkg/operator/ceph/cluster/watcher.go

```diff
@@ -91,20 +91,18 @@ func (c *clientCluster) onK8sNode(object runtime.Object) bool {
 		// If so we don't need to reconcile, this is done to avoid double reconcile on operator restart
 		// Assume the admin key since we are watching for node status to create OSDs
```

Comment:
I really wonder if we need to implement #7534 at the same time. If this event triggers a reconcile too often, we will immediately need a way to disable it.

Reply:
Okay will create a fix soon.